platforms/lambda/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 generator client {
   provider      = "prisma-client-js"
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
+  binaryTargets = ["native"]
```

Comment:
```suggestion
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
```
This is needed so generate works on native (in CI) and on "rhel-openssl-1.0.x" which is lambda